packages/invoices/src/InboundShipment/DetailView/modals/utils.ts

```diff
@@ -27,6 +27,7 @@ const createDraftInboundLine = ({
     numberOfPacks: 0,
     packSize: 0,
```

Comment:
Would you mind changing this to default to `1` - now that `0` isn't a valid value it's a bit odd that we default to it

Reply:
thanks!